### ISSUE TITLE
Type of Rooms

### ISSUE THREAD

**Issue by pikavee** [meeting-room-booking-system/mrbs-code#3753]
Is it possible to add personal type of Rooms (with different color)?
Or... where, in the code, is it possible to add other the existing two (internal or external).
Thanks

**Comment by campbell-m** [comment]
See the section on "Entry Types" in systemdefaults.inc.php.  Do not modify systemdefaults.inc.php but put any changes in config.inc.php.

**Comment by pikavee** [comment]
Thank you so much! It work!
But for the colors? Is it possibile to choose them?

**Comment by campbell-m** [comment]
Yes, you can change the colours by editing the array $color_types in the styling.inc file in the Themes directory.
